### ISSUE TITLE
Increase the display of the Month on the frontend

### ISSUE THREAD

**Issue by RachL** [orzubalsky/tradeschool#127]
Hi Or,

A feedback that I received yesterday made me understand that potential students got confused by the way we were displaying the date & month of our classes.

This is also a bit link with the translation issues: indeed, some part of the website seems to be translated correctly like the month, the day & the "pm" "am" system.

So when French people look at the date on our website, e.g. with this screenshot:
http://awesomescreenshot.com/0262dbfq53

they think the course will be held on a Tuesday ("Mardi" in French) because of the "Mar" at the front, which describe the month "March" ("Mars" in French). This is due also because in French we display the date like this "day -month - year" instead of "month-day-year".

So it's not a priority, but can we imagine to add one more letter to the display of the Month or to display the name of the month entirely to avoid confusion?

Hope that my explaination are clear. Thanks for your feedback!

**Comment by orzubalsky** [comment]
Yes, it's possible to change the way dates and times are displayed that will be specific to your school. I don't think it's possible to display 4 letter month display but there are other options. 

Can you take a look at this page https://docs.djangoproject.com/en/dev/ref/templates/builtins/#date and let me know what format would work best? If it's too confusing I can make some suggestions.

**Comment by RachL** [comment]
Hi Or,

My guess is that:
 RFC 2822 formatted date.   'Thu, 21 Dec 2000 16:01:07 +0200'

Would be the best one! 

But can the difference to Greenwich time in hours be deleted? If not, it's fine like this as well!

Many thanks!

Rachel

**Comment by Z1800000000** [comment]
This issue was moved to Z1800000000/tradeschool#26
